index.html

```diff
@@ -595,6 +601,12 @@
 
 ### Multiple National Credentials
 
+User agents should only present the minimum information
+that is necessary to answer a request.
+For example, user agents should omit
+parts of a credential that can be technically withheld,
+including information about any credentials that the user didn't select.
+
```

Comment:
This aligns with the TAG's [Design Principle](https://www.w3.org/TR/design-principles/#data-minimization) and [Privacy Principle](https://www.w3.org/TR/privacy-principles/#data-minimization) on data minimization.